docs/Current_Code.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import numpy
+from pyfirmata import Arduino, util
+import time
+from time import sleep
+from pyfirmata import INPUT, OUTPUT, PWM
+import serial
+import re
+import matplotlib.pyplot as plt
+
+class myArduinoBoard(object):
+    def __init__(self,portname = '/dev/tty.usbmodem14201'):
+        self.port = portname
+
+    def setup_arduino(self):
+        self.board = Arduino(self.port)
+        self.arduino = serial.Serial(self.port, 9600)
+        sleep(0.02)  # The synchronization between the Arduino board and pyFirmata requires some time. 1 = 1 second
+        self.analog_d9p = self.board.get_pin('d:9:p')  # Same as board.digital[9].mode = PWM
+        self.board.analog[0].mode = INPUT  # voltage
+        self.board.analog[2].mode = INPUT  # current
+        self.analog_d9p.frequency = 31372.25
```

Comment:
It is generally not a good idea to hard-code quantities deep in the code.  If this number is a fairly immutable number (rarely if ever changes) then give it a recognizable global definition, like ``D9P_FREQUENCY = 31372.25`` which is defined at the beginning of the module.